.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
```

Comment:
Is `name: CI` the best name for this workflow? Would `build.yml` and `name: Build` be clearer?

Reply:
OK, I'm cool with that. I'm also going to remove the run on `master` because we're doing the same thing with the release-on-tag workflow.